resources/i18n/en.json

```diff
@@ -374,6 +375,13 @@
                 "CONNECTOR_EDITOR": "Connector editor",
                 "JSON_EDITOR": "JSON editor"
             },
+            "DIALOG": {
```

Comment:
I think this should go into the connector-editor module, this translation keys belong there.